tools/uefi_builder/aarch64/Cargo.toml

```diff
@@ -10,7 +10,8 @@ uefi_builder_common = { path = "../common" }
 
 [dependencies.uefi-bootloader]
 artifact = "bin"
-git = "https://github.com/theseus-os/uefi-bootloader"
+git = "https://github.com/NathanRoyer/uefi-bootloader"
+branch = "aarch64-fixes"
 target = "aarch64-unknown-uefi"
```

Comment:
This is temporary, I'll leave Klim some time to review the relevant PR.

Reply:
Can be removed now :+1: